psl-clj/template/resources/leiningen/new/psl_clj/log4j.properties

```diff
@@ -0,0 +1,5 @@
+log4j.rootLogger=INFO, stdout
```

Comment:
Add in one more section for our DB connection pool:
https://github.com/linqs/psl-examples/blob/master/citation-categories/groovy/src/main/resources/log4j.properties#L13

Reply:
That's much nicer.  Done.